docs/commands/rhoas_kafka_consumer-group_reset-offset.adoc

```diff
@@ -0,0 +1,57 @@
+ifdef::env-github,env-browser[:context: cmd]
+[id='ref-rhoas-kafka-consumer-group-reset-offset_{context}']
+= rhoas kafka consumer-group reset-offset
+
+[role="_abstract"]
+Reset offset for a consumer group
+
+[discrete]
+== Synopsis
+
+Reset the offset for consumers in a consumer group reading from a given topic.
```

Comment:
We need more information. 

List offset types. 
We need to list 2 modes (partition mode is only for advanced use cases)